src/components/CreateProject/CreateProjectModal.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ModalProps } from '@mantine/core';
+import { Modal } from '@mantine/core';
+
+import { CreateProjectForm } from './CreateProjectForm';
+
+type Props = Pick<ModalProps, 'onClose' | 'opened'>;
+
+export const CreateProjectModal = ({ opened, onClose }: Props): JSX.Element => {
```

Comment:
```suggestion
export const CreateProjectModal = ({ opened, onClose }: Props) => {
```
we don't need to specify a return type, it'll be infered automaticly.